src/pytest_subtests/plugin.py

```diff
@@ -126,6 +135,8 @@ def _addSubTest(
 def pytest_configure(config: pytest.Config) -> None:
     TestCaseFunction.addSubTest = _addSubTest  # type: ignore[attr-defined]
     TestCaseFunction.failfast = False  # type: ignore[attr-defined]
+    TestCaseFunction._originaladdSkip = copy.copy(TestCaseFunction.addSkip)  # type: ignore[attr-defined]
+    TestCaseFunction.addSkip = _addSkip  # type: ignore[attr-defined]
```

Comment:
The `_addSkip` in `unittest.case` doesn't contain logic to call `addSubTest` for `outcome.skipped` (unlike `_feedErrorsToResult`), and it turns out this causes the reporting issue described in the PR.

Unlike `addSubTest` (where original `TestCaseFunction` doesn't have it), `addSkip` is defined `TestCaseFunction`, and here we save the original implementation and patch `TestCaseFunction.addSkip` to a newly defined `_addSkip` that allows us to call `addSubTest`.

The `_originaladdSkip` allows us to restore to it in `pytest_unconfigure` as well as it turns out necessary to call it too inside the newly defined `_addSkip`.